src/main/java/uk/gov/hmcts/reform/blobrouter/services/email/EmailSender.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.blobrouter.services.email;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.util.Map;
+import javax.mail.internet.MimeMessage;
+
+@Component
+@ConditionalOnProperty(prefix = "spring.mail", name = "host")
```

Comment:
probably i'm missing the big picture or those zips in diff are truly blocking me.. i don't see any config change. perhaps we don't need any components for now nor conditions. otherwise i would enquire integration test to assure email is never sent on any environment other than prod

Reply:
zip files are for testing only. They are really not zip.
I will deal with integration testing, I agree unit tests are not enough.